qa/tasks/kafka.py

```diff
@@ -108,19 +112,11 @@ def run_kafka(ctx,config):
         for (client, _) in config.items():
             (remote,) = ctx.cluster.only(client).remotes.keys()
 
-            toxvenv_sh(ctx, remote, 
-                ['cd', '{tdir}/bin'.format(tdir=get_kafka_dir(ctx, config)), run.Raw('&&'),
-                 './kafka-server-stop.sh',  
-                 '{tir}/config/kafka.properties'.format(tir=get_kafka_dir(ctx, config)),
-                ]
-            )
-
-            toxvenv_sh(ctx, remote, 
-                ['cd', '{tdir}/bin'.format(tdir=get_kafka_dir(ctx, config)), run.Raw('&&'), 
-                 './zookeeper-server-stop.sh',
-                 '{tir}/config/zookeeper.properties'.format(tir=get_kafka_dir(ctx, config)),
-                ]
-            )
+            remote.run(
```

Comment:
are you sure this is needed?
isn't the change in line 58 enough (together with proper shutdown)?

Reply:
The change at line 58 is removing the kafka-2.6.0-src/logs. So, probably this is required.

Reply:
the change in line 58 is required.
however, I'm not sure we should use `kill -9` here. probably better to use the proper "close" scripts

Reply:
The close scripts were giving some failures (as discussed), so I used this option.

Reply:
did you try the combination of the close scripts and the deletion of the logs directory?

Reply:
I've scheduled a run for that.

Reply:
The combination of close scripts and logs directory deletion worked now: https://pulpito.ceph.com/kapandya-2021-04-07_19:03:41-rgw:notifications-wip-rgw-notifications-distro-basic-smithi/